ios-patches/003_support_ios8.diff

```diff
@@ -0,0 +1,91 @@
+From e41d51094a99afea3d11bc2a8e6a36725d12e709 Mon Sep 17 00:00:00 2001
+From: Michael Kirk <michael.code@endoftheworl.de>
+Date: Sun, 11 Feb 2018 15:21:38 -0800
+Subject: [PATCH] iOS8 compatibility
+
+---
+ sdk/objc/Framework/Classes/Common/helpers.mm                     | 3 ++-
+ sdk/objc/Framework/Classes/Metal/RTCMTLVideoView.m               | 9 ++++++++-
+ .../Framework/Classes/PeerConnection/RTCCameraVideoCapturer.m    | 3 ++-
+ .../Classes/Video/RTCAVFoundationVideoCapturerInternal.mm        | 3 ++-
+ tools_webrtc/ios/build_ios_libs.py                               | 2 +-
+ 5 files changed, 15 insertions(+), 5 deletions(-)
+
+diff --git a/sdk/objc/Framework/Classes/Common/helpers.mm b/sdk/objc/Framework/Classes/Common/helpers.mm
+index 4afa15d11..5876ee86e 100644
+--- a/sdk/objc/Framework/Classes/Common/helpers.mm
++++ b/sdk/objc/Framework/Classes/Common/helpers.mm
+@@ -120,7 +120,8 @@ int GetProcessorCount() {
+ #if defined(__IPHONE_9_0) && defined(__IPHONE_OS_VERSION_MAX_ALLOWED) \
+     && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_9_0
+ bool GetLowPowerModeEnabled() {
+-  if (isOperatingSystemAtLeastVersion(9.0)) {
++  //if (isOperatingSystemAtLeastVersion(9.0)) {
++  if (@available(iOS 9, *)) {
```

Comment:
Even though logically equivalent to the replaced code, xcode expects you to use `@available` for checks like this, else it's static analysis produces warning which will fail the build.